src/platform-aws.c

```diff
@@ -424,6 +424,25 @@ int platform_init(void)
 			goto exit;
 		}
 	}
+
+	/*
+	 * NCCL v2.19.3 reduced the chunk size used when running NVLS Tree
```

Comment:
Prefer to put this context also in the commit message body itself.
In addition, I prefer that we won't only provide link to the github PR (https://github.com/NVIDIA/nccl/pull/1112) but also provide here the data of the performance diff we see when this is applied on NCCL v2.20.3.

Reply:
Ack, updating.